### ISSUE TITLE
The link to stringsjs.com takes you to another site

### ISSUE THREAD

**Issue by juggledad** [Steveorevo/node-red-contrib-string#11]
http://stringjs.com/ no longer exists. this takes you to another site

**Comment by Steveorevo** [comment]
Yes, sadly that domain has expired; but that's what that way back Time Machine is fore! :) 

https://web.archive.org/web/20171201072159/http://stringjs.com/

Or his github repo still lives at:

https://github.com/jprichardson/string.js

**Comment by juggledad** [comment]
You might want to change the link in the description

**Comment by Steveorevo** [comment]
Done!
